### PR TITLE
Create 2022-05-ECP22-BOF.md

### PR DESCRIPTION
**Pull request by karbarz**
Created event for this year's ECP BOF days. 

## PR checklist for files displayed on bssw.io site

* [x] ***[Author]*** `@mention` the BSSw.io editorial board member `@<eb-member-id>` in **Description** above assigned to shepherd your PR.
* [x] ***[Author]*** Add the `<issue-id>` in the **Description** above for the associated GitHub Issue.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py])~~.
* [x] ***[Author]*** Inspect the content in the `*.md` file(s) as rendered in GitHub for this PR.
* [x] ***[EB Mem]*** Assign this PR to the EB member `<eb-member-id>`.
* [x] ***[EB Mem]*** Assign this PR to the author of the PR `<pr-author-id>`.
* [x] ***[EB Mem]*** Add one or more Reviewers.
* [x] ***[EB Mem]*** Add label `content: <content-type>` for the type of contribution.
* [x] ***[EB Mem]*** Add to Project `Content Development` (see [Content Development]).
* [x] ***[EB Mem]*** Add [meta-data] to the `*.md` file(s) (set `Publish: yes`).
* [x] ***[EB Mem]*** Add label `preview` (so PR branch will be merged to 'preview' branch and watch for possible merge failures).
* [x] ***[EB Mem]*** Rebuild [preview] site and confirm new content is there, renders correctly and is returned in searches.
* [ ] ***[Author]*** Make any final changes to the PR based on feedback.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py]).~~
* [ ] ***[EB Mem]*** Rebuild [preview] site and re-confirm content looks correct.
* [ ] ***[EB Mem]*** Ensure at least one reviewer signs off on the final changes.
* [ ] ***[EB Mem]*** Move the PR to "Ready to Publish" in [Content Development].
* [ ] ***[EB Mem]*** Leave comment and @mention `@betterscientificsoftware/bssw-maint` (**BSSw Maint**) asking to carry out final publication steps.
* [ ] ***[BSSw Maint]*** Verify that all needed files are present in the PR (article, images, updates to Site/Homepage.md carousel and/or Site/Announcements/Announcements.md as appropriate).
* [ ] ***[BSSw Maint]*** Merge PR. (Should automatically move to "Done" in [Content Development].)
* [ ] ***[BSSw Maint]*** Verify new contribution shows up on [bssw.io] as expected.

NOTE:
* Checklist items prefixed with ***[Author]*** are expected to be performed by the author of the PR or can be performed by the author.
* Checklist items prefixed with ***[EB Mem]*** must be performed by a [BSSw.io Editorial Board (EB) Member](https://betterscientificsoftware.github.io/bssw.io/bssw_members.html).
* Checklist items prefixed with ***[BSSw Maint]*** must be performed by someone in the `@betterscientificsoftware/bssw-maint` team (hint: type `@`, `b`, `s`, `s`, `w`, `-`, `m`  to auto-complete to `@betterscientificsoftware/bssw-maint`).

### CODE REVIEW

File: Events/2022-05-ECP22-BOF.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# 2022 Exascale Computing Project Community BOF Days
+
+#### Contributed by BSSw.io Team
+
+- Date: May 10 - 12, 2022
```
Comment (bernhold) [inline Events/2022-05-ECP22-BOF.md:5]: We now prefer to say "Event Dates" (or singular, as appropriate)

Comment (karbarz) [reply Events/2022-05-ECP22-BOF.md]: Thanks, I've made that change!

File: Events/2022-05-ECP22-BOF.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Event Name | [ECP BOF Days](https://www.exascaleproject.org/event/ecp-community-bof-days-2022/) 
+Event Date & Time | May 10 - 12, 2022 (60- to 90-minute sessions throughout the day)
+Website | 	<https://www.exascaleproject.org/event/ecp-community-bof-days-2022/>  
+
+The Exascale Computing Project (ECP) 2022 Community Birds-of-a-Feather (BOF) Days will take place Tuesday, May 10 – Thursday, May 12, with six to eight sessions per day, similar to the 2021 event. 
+The 2022 Community BOF Days will provide an opportunity for the high-performance computing community to engage with ECP teams to discuss our latest development efforts. 
+Each BOF will be a 60- to 90-minute session on a given topic, with a brief overview, followed by Q&A. 
+The BOFs will be conducted via Zoom.
+
+Topics, listed below, will range from (**please put information here**), as well as general topics focusing on software sustainability and productivity, approaches to software quality assurance, etc.
```
Comment (bernhold) [inline Events/2022-05-ECP22-BOF.md:23]: I don't want to try to summarize the other topics in a couple of words -- too diverse.  Instead, I suggest switching it around to say "BOFs will include general topics focusing on... as well as a range of others related to ECP projects and acivities."

File: Events/2022-05-ECP22-BOF.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+# 2022 Exascale Computing Project Community BOF Days
+
+#### Contributed by BSSw.io Team
+
+- Date: May 10 - 12, 2022
+- Location: Online
+- Event Website: https://www.exascaleproject.org/event/ecp-community-bof-days-2022/
+- Organizers: Exascale Computing Project Community
```
Comment (bernhold) [inline Events/2022-05-ECP22-BOF.md:8]: I would drop "Community"

File: Events/2022-05-ECP22-BOF.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# 2022 Exascale Computing Project Community BOF Days
+
+#### Contributed by BSSw.io Team
```
Comment (bernhold) [inline Events/2022-05-ECP22-BOF.md:3]: You could list yourself here.

**Review by bernhold — APPROVED** [review]
I've made some localized comments.  Let me know if you have any questions about them.  Otherwise, looks good.  Thanks for putting it together!

**Review by bernhold — APPROVED** [review]
Thanks for addressing my suggestions.  It all LGTM.

**Pull request merged**